RoslynSecurityGuard.Vsix/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.AspNet.Identity.Core" version="2.2.1" targetFramework="net452" />
```

Comment:
Removed the dependance, this package wasn't needed. It was only some left-overs from earlier testing.